RecoMET/METFilters/plugins/EcalDeadCellDeltaRFilter.cc

```diff
@@ -81,30 +82,17 @@
 
 #include "DataFormats/Math/interface/deltaR.h"
 
-// HCAL
-#include "CondFormats/HcalObjects/interface/HcalChannelQuality.h"
-#include "CondFormats/DataRecord/interface/HcalChannelQualityRcd.h"
-#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
-#include "DataFormats/HcalDetId/interface/HcalDetId.h"
-
 #include "DataFormats/Provenance/interface/RunLumiEventNumber.h"
 
-#include "TFile.h"
-#include "TTree.h"
-#include "TH1.h"
-
-class EcalDeadCellDeltaRFilter : public edm::EDFilter {
+class EcalDeadCellDeltaRFilter : public edm::stream::EDFilter<> {
```

Comment:
can you confirm that it was tested to be thread-safe now after these changes? I see some TH1F was removed, which indicates that beforehand it may not have been safe.

Reply:
The TH1F was removed because it was actually never filled. With it gone, I think making the filter a stream module is possible but it was not tested. The module does not seem to be used in any configuration at the moment.